ci/cases/pr/C48_atm3DVar_ocnhybDA.yaml

```diff
@@ -0,0 +1,18 @@
+experiment:
+  system: gfs
+  mode: cycled
+
+arguments:
+  pslot: ${pslot}
+  app: S2S
+  resdet: 48
+  comrot: ${RUNTESTS}/COMROT
+  expdir: ${RUNTESTS}/EXPDIR
+  # icsdir: ${ICSDIR_ROOT}/C48cpldDA
+  icsdir: "/scratch2/NCEPDEV/ocean/Guillaume.Vernieres/data/ICSDIR/C48O500" # TODO: make platform agnostic
```

Comment:
I think this is what we already have as C48C48mx500

Reply:
ok sounds good.  I'll do a comparison and update.

Reply:
this file is removed.